doc/CONTRIBUTING.md

```diff
@@ -327,3 +327,31 @@ See the
 for more configuration details. Everything within `python_config` is imported
 via `dictConfig` (in addition to some standard settings defined in
 `dataactcore.logging`.
+
```

Comment:
Thanks for taking the time to add this---much appreciated. Have clear guidance for using the loggers in our codebase is helpful.